pkgs/applications/editors/vscode/vscodium.nix

```diff
@@ -34,6 +34,7 @@ in
     version = "1.57.1";
     pname = "vscodium";
 
+    sourceExecutableName = "codium";
     executableName = "codium";
```

Comment:
isn't this a duplicate with executableName?

Reply:
Not for code-insiders... Maybe add a default value to set this to `executableName`?

Reply:
The variables may hold the same value (or not), but their purpose is very different. `executableName` is what we want the output to be, so you can call `$ codium`, whereas `sourceExecutableName` is governed entirely by Microsoft, or whoever generates the archive we download.

But I think setting it to `executableName` per default is a good idea 👍

Reply:
Fixed